app/views/api/v2/hosts/main.json.rabl

```diff
@@ -26,6 +26,10 @@ attributes :configuration_status => :puppet_status
 # to avoid renaming model_name to match accessors
 attributes :hardware_model_name => :model_name
 
+node :compute_resource_provider do |host|
+  host.compute_resource&.provider&.downcase
```

Comment:
I don't have an environment at hand to check, but does this doesn't result in additional DB queries, correct?

Reply:
How to do the checking? Any suggestion?

Reply:
It does not add additional DB queries.

Reply:
Great. Thanks for verifying. Those are the kind of issues where it appears to work, but can cause a lot of issues for large deployments.